openbas-front/src/admin/components/simulations/simulation/timeline/TimelineOverview.tsx

```diff
@@ -115,7 +115,8 @@ const TimelineOverview = () => {
 
   const filteredInjects = filtering.filterAndSort(injects);
 
-  const pendingInjects = filtering.filterAndSort(injects.filter((i: InjectStore) => i.inject_status === null));
+  const isNotDisable = (inject: InjectStore): boolean => inject.inject_injector_contract.injector_contract_content_parsed?.config.expose && inject.inject_enabled;
```

Comment:
Double negative variable name. While I understand why, wouldn't isEnabled be better still?